index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css">
+</head>
+<body style=" background-color: #a5d3eb;">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div class="container">
+        <div class="row">
+            <main id="main" class="text-center">
```

Comment:
This div begins outside the main section and finish within it

Reply:
In this case, you pointed out the problem with `div` but forget to remind him that `what he should do and why`. Please consider reviewing your suggestion.

Reply:
- [ ] Great job working on the suggested changes! 👏🏼 However, there's still some room to improve. Please avoid justifying your changes by saying that it is "best practice" as it is too vague of a reason. Please provide a better one so the person receiving the review can learn why the change could improve its code. Think of: how fixing the HTML structure could improve the codebase. 🤓 🔧